launcher/ui/pages/global/LauncherPage.cpp

```diff
@@ -141,6 +143,25 @@ void LauncherPage::on_instDirBrowseBtn_clicked()
                 ui->instDirTextBox->setText(cooked_dir);
             }
         }
+        else if(Flatpak::IsFlatpak() && raw_dir.startsWith("/run/user"))
+        {
+            QMessageBox warning;
+            warning.setText(tr("You're trying to specify an instance folder "
+                            "which was granted temporaily via Flatpak.\n"
+                            "This is known to cause problems, "
+                            "after a restart the launcher might break, "
+                            "because it will no longer have access to that directory.\n\n"
+                            "Granting PolyMC access to it via Flatseal is recommended."));
+            warning.setInformativeText(
+             tr("Do you really want to use this path?\n"
+             "Selecting \"No\" will close this and not alter your instance path."));
+            warning.setStandardButtons(QMessageBox::Yes | QMessageBox::No);
+            int result = warning.exec();
+            if (result == QMessageBox::Yes)
+            {
+                ui->instDirTextBox->setText(cooked_dir);
+            } 
```

Comment:
```suggestion
            QMessageBox warning;
            warning.setText(tr("You're trying to specify an instance folder "
                            "which was granted temporaily via Flatpak.\n"
                            "This is known to cause problems. "
                            "After a restart the launcher might break, "
                            "because it will no longer have access to that directory.\n\n"
                            "Granting PolyMC access to it via Flatseal is recommended."));
            warning.setInformativeText(
             tr("Do you want to proceed anyway?"));
            warning.setStandardButtons(QMessageBox::Ok | QMessageBox::Cancel);
            int result = warning.exec();
            if (result == QMessageBox::Ok)
            {
                ui->instDirTextBox->setText(cooked_dir);
            } 
```

Make this a *Ok-Cancel* messagebox instead

Reply:
alter previous instance folder alert too? to make it consistent